python/cuvs_bench/cuvs_bench/tests/test_registry.py

```
@@ -326,40 +329,55 @@ class TestBackendIntegration:
-        result = backend.build(
-            dataset=dataset,
-            build_params={"nlist": 1024},
-            index_path=Path("/tmp/test_index"),
-        )
+        indexes = [
+            IndexConfig(
+                name="test_index",
+                algo="dummy_algo",
+                build_param={"nlist": 1024},
+                search_params=[{"nprobe": 10}],
+                file="/tmp/test_index",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_index"

(S108)